lib/angular2/shared/storage/storage.swaps.ts

```diff
@@ -6,7 +6,7 @@
  * The InternalStorage class is used for dependency injection swapping.
  * It will be provided using factory method from different sources.
  **/
-export class Storage {
+export class BaseStorage {
```

Comment:
Why is this change part of this pull request? it seems it squashed commits non related to this functionality.. This modifies the history